esmvaltool/cmorizers/obs/cmor_config/CDS-SATELLITE-ALBEDO.yml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Is this file used for anything?

Reply:
Yes, but it should not be in this branch.. I am taking it out of this PR.